VideoLocker/src/main/java/org/edx/mobile/module/notification/UserNotificationManager.java

```diff
@@ -67,10 +67,14 @@ public void checkAppUpgrade(){
             return;
         PrefManager.AppInfoPrefManager pmanager =
                 new PrefManager.AppInfoPrefManager(MainApplication.instance());
-        if ( pmanager.isAppUpgradeNeedSyncWithParse() ){
-            pmanager.setAppUpgradeNeedSyncWithParse(false);
-            resubscribeAll();
-        }
+        //TODO - not sure if we need to resync with parse server.
```

Comment:
So are you planning to potentially uncomment this once you figure out what's going on?